presto-cli/src/main/java/io/prestosql/cli/ClientOptions.java

```diff
@@ -150,10 +150,12 @@
     {
         ALIGNED,
         VERTICAL,
-        CSV,
         TSV,
-        CSV_HEADER,
         TSV_HEADER,
+        CSV,
+        CSV_HEADER,
+        CSV_WITHOUT_QUOTES,
+        CSV_HEADER_WITHOUT_QUOTES,
```

Comment:
We could shorten the names to make it easier to type. Comparison of some names:
* `CSV_WITHOUT_QUOTES`
* `CSV_NO_QUOTES`
* `CSV_UNQUOTED`